src/style.css

```diff
@@ -0,0 +1,66 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+body * {
+  padding: 0;
+  font-family: Avenir, serif;
+}
```

Comment:
Try to avoid selector *. Yes in these tasks it doesn't influence performance, but in bigger projects, it can load browser

Reply:
apply font directly to input, not to all inside body

Reply:
Try to avoid increasing the specificity of the selector. One class is enough to identify the elements on the page. No need to write something like .block.colored-block. Use .block and .block--colored instead, and apply each of them to your element